lib/solidus_social/config.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'solidus_social/social_configuration'
+
+module Spree
```

Comment:
No need to wrap within the `Spree` module. I'd just add the `::Spree::SocialConfig` assignment in the `social_configuration.rb` file.

Reply:
Modified this here [36f081b](https://github.com/solidusio-contrib/solidus_social/commit/36f081b55005ad895a564c8c2cf34e63cb63b325)

Fixed now